src/test/java/seedu/address/storage/JsonAdaptedMemberTest.java

```diff
@@ -46,6 +47,12 @@ public void toModelType_validMemberDetails_returnsMember() throws Exception {
         assertEquals(BENSON, member.toModelType());
     }
 
+    @Test
```

Comment:
Maybe rename to toModelType_validMemberDetailsWithNoOptionalFields_returnsMember() instead of using number 2